components/popover/index.css

```diff
@@ -14,94 +14,110 @@ governing permissions and limitations under the License.
-
-  background-color: var(--mod-popover-background-color, var(--spectrum-popover-background-color));
-  filter: var(--mod-popover-filter, var(--spectrum-popover-filter));
-
-  /* has tip triangle */
-  &.spectrum-Popover--withTip {
-    .spectrum-Popover-tip {
-      /* triangle polygon */
-      .spectrum-Popover-tip-triangle {
-        stroke-linecap: square;
-        stroke-linejoin: miter;
-        fill: var(--highcontrast-popover-background-color, var(--mod-popover-background-color, var(--spectrum-popover-background-color)));
-        stroke: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color)));
-        stroke-width: var(--mod-popover-border-width, var(--spectrum-popover-border-width));
-      }
-    }
-  }
+	--spectrum-popover-filter: drop-shadow(var(--mod-popover-drop-shadow-x, var(--spectrum-popover-drop-shadow-x)) var(--mod-popover-drop-shadow-y, var(--spectrum-popover-drop-shadow-y)) var(--mod-popover-drop-shadow-blur, var(--spectrum-popover-drop-shadow-blur)) var(--mod-popover-drop-shadow-color, var(--spectrum-popover-drop-shadow-color)));
+	@extend %spectrum-overlay;
+
+	box-sizing: border-box;
+	position: absolute;
+
+	outline: none; /* Hide focus outline */
+
+	padding: var(--mod-popover-content-area-spacing, var(--spectrum-popover-content-area-spacing));
+
+	display: inline-flex;
+	flex-direction: column;
+
+	border-style: solid;
+	border-radius: var(--mod-popover-corner-radius, var(--spectrum-popover-corner-radius));
+	border-color: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color-with-transparency)));
+
+	/* border-color: var(--spectrum-popover-border-color-default); */
+
+	border-width: var(--mod-popover-border-width, var(--spectrum-popover-border-thickness));
+
+	background-color: var(--mod-popover-background-color, var(--spectrum-popover-background-color));
+	filter: var(--mod-popover-filter, var(--spectrum-popover-filter));
+
+	/* has tip triangle */
+	&.spectrum-Popover--withTip {
+		.spectrum-Popover-tip {
+			/* triangle polygon */
+			.spectrum-Popover-tip-triangle {
+				stroke-linecap: square;
+				stroke-linejoin: miter;
+				fill: var(--highcontrast-popover-background-color, var(--mod-popover-background-color, var(--spectrum-popover-background-color)));
+				stroke: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color-with-transparency)));
+				stroke-width: var(--mod-popover-border-width, var(--spectrum-popover-border-thickness));
+			}
+		}
+	}
 }
 
 /* default opens and animates upward */
 .spectrum-Popover.is-open {
-  @extend %spectrum-overlay--open;
+	@extend %spectrum-overlay--open;
 }
 
 /* prevent nested popovers from affecting each others drop shadow filters */
 .spectrum-Popover * {
-  --mod-popover-filter: none;
+	--mod-popover-filter: none;
+
+	& > .spectrum-Popover {
+		margin: var(--spectrum-popover-nested-container-spacing);
```

Comment:
This was my attempt at trying to incorporate the "container padding" that's outlined in Figma. I'm still unsure if this component (the popover itself) is really the right place for that "container padding." Should it go on that popover wrapper we have instead?

https://github.com/adobe/spectrum-css/blob/61af0743f874d799b8fbb789aff8ad2580c5994a/components/popover/stories/template.js#L165

Reply:
/* & > .spectrum-Popover {
		margin: var(--spectrum-popover-nested-container-spacing);
	} */